services/buildlogs/filter.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"github.com/finitum/AAAAA/pkg/models"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+func FilterJobs(jobs []models.Job, nameFilter, statusFilter, start, sortKey, limit string) ([]models.Job, error) {
```

Comment:
What's the purpose of this function, comments?

Reply:
Added comments and improved API in latest commit